site/public/subscribe.html

```diff
@@ -0,0 +1,139 @@
+   <div class="silk-surface silk-smooth-color kobweb-box kobweb-align-top-start" style="min-height: 100vh;">
+    <div class="kobweb-box kobweb-align-top-start" style="width: 100%; height: 100%;">
+     <div class="kobweb-col kobweb-arrange-top kobweb-align-start" style="width: 100%; height: 100%;">
+      <div class="conditional-layout-mobile">
+       <div class="shared-navbar-styles kobweb-row kobweb-arrange-spaced-by kobweb-arrange-start kobweb-align-center-vert" style="--kobweb-arrange-spaced-by: 16px;">
+        <img class="silk-image" style="cursor: pointer;" src="/icons/hamburger.svg" alt="Abrir menu">
+        <div class="kobweb-row kobweb-arrange-start kobweb-align-center-vert">
+         <img class="silk-image" style="display: inline-block; width: 40px; height: 40px; margin: 0px 16px 0px 0px;" src="/logo.svg" alt=""><span class="silk-span-text" style="font-size: 14px; color: black; font-weight: 800; text-decoration-line: none;">Android Dev Peru</span>
+        </div>
+       </div>
+       <div style="height: 48px;"></div>
+      </div>
+      <div class="conditional-layout-large">
+       <div class="shared-navbar-styles kobweb-row kobweb-arrange-space-between kobweb-align-top">
+        <a style="height: 100%; display: flex; cursor: pointer; text-decoration-line: none;" href="/">
+         <div class="kobweb-row kobweb-arrange-start kobweb-align-center-vert">
+          <img class="silk-image" style="display: inline-block; width: 40px; height: 40px; margin: 0px 16px 0px 0px;" src="/logo.svg" alt=""><span class="silk-span-text" style="font-size: 14px; color: black; font-weight: 800; text-decoration-line: none;">Android Dev Peru</span>
+         </div></a>
+        <div class="kobweb-row kobweb-arrange-start kobweb-align-top" style="height: 100%;">
+         <a class="silk-link navbar-large-item nav-link" href="/">Inicio</a><a class="silk-link navbar-large-item nav-link" href="/about-us">Quienes somos</a><a class="silk-link navbar-large-item nav-link" href="/resources">Recursos</a>
+        </div>
+       </div>
+       <div style="height: 48px;"></div>
+      </div>
+      <div class="main-site">
+       <div style="padding: 2em;">
+        <div style="display: flex; justify-content: center;">
+         <form action="https://app.convertkit.com/forms/6923083/subscriptions" style="background-color:#fff;border-radius:6px" class="seva-form formkit-form" method="post" data-sv-form="6923083" data-uid="f4a99c32b5" data-format="inline" data-version="5" min-width="400 500 600 700">
```

Comment:
Ok, this is bad news... looks like my attempt to load the `<script>` tag programmatically, as recommended by ConvertKit, didn't work as expected. **TL;DR;** This means that any time the form changes, we'll have to re-publish the site. Not ideal but this will do it for now...

_Deep dive:_

Looks like my code is indeed adding the script tag correctly. However, the exported static site, which uses Playwright to actually render the HTML, is picking up the HTML-state _after_ the script is run and executed. This causes the content to be static, rather than dynamically loaded every time with the script. This means that any time the form changes, we'll have to re-publish the site. Not ideal but this will do it for now...